extension.js

```diff
@@ -88,9 +78,9 @@ class Extension {
       return;
     }
 
-    if (this.extWm) this.extWm.disable();
+    this.extWm?.disable();
```

Comment:
I suppose, future Forge development starts with conciseness versus readability. On the fence on this one.

Reply:
I didn't know or too lazy to read new features/new JS operators. I am also OK with this.